src/machine/usb/usb.go

```diff
@@ -75,7 +79,9 @@ const (
 	HID_ENDPOINT_OUT  = 5 // for Interrupt Out
 	MIDI_ENDPOINT_IN  = 6 // for Bulk In
 	MIDI_ENDPOINT_OUT = 7 // for Bulk Out
-	NumberOfEndpoints = 8
+	MSC_ENDPOINT_IN   = 8 // for Bulk In
+	MSC_ENDPOINT_OUT  = 9 // for Bulk Out
+	NumberOfEndpoints = 10
```

Comment:
This could be problematic for other microcontrollers. For example, the atsamd21, atsamd51, and nrf52840 only have a total of 8 endpoints. Some sort of conditional branching might be necessary.